src/zenml/models/v2/core/artifact.py

```diff
@@ -286,6 +346,14 @@ class ArtifactFilter(WorkspaceScopedFilter):
         default=None,
         description="Name of the artifact",
     )
+    version: Optional[str] = Field(
```

Comment:
Agreed, but I think filtering by tags should be done in a separate PR since there are also other entities that have tags which could benefit from this

Reply:
Before we resolve this it might be nice to create a ticket for it then.